examples/release_highlights/plot_release_highlights_1_2_0.py

```diff
@@ -0,0 +1,79 @@
+# flake8: noqa
+"""
+=======================================
+Release Highlights for scikit-learn 1.2
+=======================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 1.2! Many bug fixes
+and improvements were added, as well as some new key features. We detail
+below a few of the major features of this release. **For an exhaustive list of
+all the changes**, please refer to the :ref:`release notes <changes_1_2>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install -c conda-forge scikit-learn
+
+"""
+
+# %%
+# Pandas output with `set_output` API
+# -----------------------------------
+# scikit-learn's transformers now support pandas output with the `set_output` API.
+# See :ref:`sphx_glr_auto_examples_miscellaneous_plot_set_output.py`, to learn more
+# about how to use the API.
+import numpy as np
+from sklearn.datasets import load_iris
+from sklearn.preprocessing import StandardScaler, KBinsDiscretizer
+from sklearn.compose import ColumnTransformer
+
+X, y = load_iris(as_frame=True, return_X_y=True)
+sepal_cols = ["sepal length (cm)", "sepal width (cm)"]
+petal_cols = ["petal length (cm)", "petal width (cm)"]
+
+preprocessor = ColumnTransformer(
+    [
+        ("scalar", StandardScaler(), sepal_cols),
+        ("kbin", KBinsDiscretizer(encode="ordinal"), petal_cols),
+    ],
+    verbose_feature_names_out=False,
+)
+preprocessor.set_output(transform="pandas")
+
+X_out = preprocessor.fit_transform(X)
+X_out.sample(n=5, random_state=0)
+
```

Comment:
What about linking to or even embedding @amueller's video here?

https://www.youtube.com/watch?v=J4KCu9WWDTo

Reply:
That makes sense.  I suspect there is a privacy concern with embedding a Youtube video.

I added a link to Andy's video to the highlights.

Reply:
@thomasjpfan 
Can we link to this video instead?  We updated it so the sound was easier to hear:  
https://youtu.be/5bCg8VfX2x8

Reply:
Thanks. Updated in [`06c065b` (#24798)](https://github.com/scikit-learn/scikit-learn/pull/24798/commits/06c065b4c2ed2b9875cf412aaf85a466cb6cb0ab)